### PR TITLE
Set the properties of a Set node after add_node

### LINKED ISSUE #32718: VS Vector3 index
**Godot version:**
Godot 3.2 Alpha Official Release

**OS/device including version:**
Windows 10

**Issue description:**
In Visual Script when setting Vector3 (and other such composed ones like Vector2, transform etc) fail to show index option in inspector untill "assign op" is changed from assign to some other, works fine in assign after it was changed and put back to assign

**Steps to reproduce:**
1.Create any Setter that sets composed variables such as Vectors
2.Click on them - Inspector shows no index
3.Change "Assign Op" in the inspector to any other option
4.Index Appears
5.Set "Assign Op" to what it was
6.Index is still there

**Minimal reproduction project:**
[Project.zip](https://github.com/godotengine/godot/files/3713464/Project.zip)

### PR DESCRIPTION
**Pull request by OsamaElHariri**
Closes #32718

### CODE REVIEW

File: modules/visual_script/visual_script_editor.cpp [hunk 1]
```diff
@@ -3515,6 +3515,7 @@ void VisualScriptEditor::_selected_connect_node(const String &p_text, const Stri
 	}
 
 	Ref<VisualScriptNode> vnode;
+	Ref<VisualScriptPropertySet> scriptPropSet;
```
Comment (akien-mga) [inline modules/visual_script/visual_script_editor.cpp]: We use `snake_case` for identifiers, so this should be `script_prop_set`.

**Comment by akien-mga** [comment]
CC @fire @swarnimarun

**Comment by swarnimarun** [comment]
The code looks fine to me, I will test the code just in case and report back.

File: modules/visual_script/visual_script_editor.cpp (hunk 1 above)
Comment (OsamaElHariri) [reply modules/visual_script/visual_script_editor.cpp]: Woops, my bad. Fixed this :+1:  @akien-mga

**Comment by swarnimarun** [comment]
Works fine on my side. :+1:

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**